dll/win32/newdev/wizard.c

```diff
@@ -579,6 +579,33 @@ AutoDriver(HWND Dlg, BOOL Enabled)
     IncludePath(Dlg, Enabled & IsDlgButtonChecked(Dlg, IDC_CHECK_PATH));
 }
 
+static INT CALLBACK
+BrowseCallbackProc(HWND hwnd, UINT uMsg, LPARAM lParam, LPARAM lpData)
+{
+    switch (uMsg)
+    {
+        case BFFM_INITIALIZED:
+        {
+            LPCWSTR pszPath = ((PDEVINSTDATA)lpData)->CustomSearchPath;
+            SendMessageW(hwnd, BFFM_SETSELECTION, TRUE, (LPARAM) pszPath);
+            SendMessageW(hwnd, BFFM_ENABLEOK, 0, CheckBestDriver(NULL, (LPWSTR)pszPath));
```

Comment:
The first parameter to the CheckBestDriver call cannot be NULL (check how it's used in the functions).